docs/api-docs/storefront/graphql/graphql-product-reviews.mdx

```diff
@@ -0,0 +1,191 @@
+# Product Reviews with the GraphQL Storefront API
+
+BigCommerce's GraphQL Storefront API allows you to create and query [product reviews](https://support.bigcommerce.com/s/article/Managing-Reviews) through the built-in comment system. 
+
+
+## Authentication
+
+To work with product reviews, the [API account](/docs/start/authentication/api-accounts) that generates [Storefront tokens](/docs/rest-authentication/tokens) must include one of the following [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes):
+
+| UI Name | Permission | Parameter |
+|:--------|:-----------|:----------|
+| Content | modify | `store_v2_content` |
```

Comment:
Is this accurate? I tested this OAuth scope briefly

Reply:
API Account scopes are not relevant to making GQL requests. Once you have a valid Storefront JWT token you will be able to utilize the add product review mutation.

Reply:
@bc-jz Do you have the answer for this? Or I might remove

Reply:
I think it should be removed.

Reply:
Yes, this is the scope needed to generate a JWT. If you know the scope, it would be great if we could include it

Reply:
Ah I see, I think I'll remove it